servicebroker/lib/util/messageUtil.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var path = require('path');
+var fs = require('fs');
+var catalog = JSON.parse(
+  fs.readFileSync(path.join(__dirname, '../../config/catalog.json'), 'utf8'));
+module.exports = function(){
+  var messageUtil = {};
+  var message = {
+    POLICY_REQUIRED: "Policy is required as a parameter for binding request.",
+    INVALID_POLICY_FORMAT: "Invalid policy format.",
+    DUPLICATED_BIND: "Another " + catalog.services[0].name + " service instance already bound to current application ${applicationId}. The multiple binding is not supported.",
```

Comment:
DUPLICATE_BIND: "A" + catalog.services[0].name + " service instance is already bound to application ${applicationId}. Multiple bindings are not supported.",